informer_crons.go

```diff
@@ -10,6 +10,9 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+var checkinMargin int64 = 1
+var maxRunTimeSlack int64 = 3
```

Comment:
Let's rename these to indicate that these are default values. FWIW, we can even drop these for now, and rely on the SDK/backend defaults (unless you discovered we really need them?)

For later: we should let users set Sentry-specific annotations on the CronJob resource, so that people can control these parameters.